vllm/engine/arg_utils.py

```diff
@@ -645,33 +614,16 @@ def create_engine_config(self, ) -> EngineConfig:
             raise ValueError(
                 "BitsAndBytes load format and QLoRA adapter only support "
                 f"'bitsandbytes' quantization, but got {self.quantization}")
-        if self.image_input_type:
-            if (not self.image_token_id or not self.image_input_shape
-                    or not self.image_feature_size):
+        if self.image_token_id:
```

Comment:
I think we should explicitly check against `None` here (in case `image_token_id=0` for some odd reason)